website/views.py

```diff
@@ -191,23 +192,27 @@ def get_dashboard_nodes(auth):
     """
     user = auth.user
 
-    contributed = user.node__contributed  # nodes user contributed to
-
-    nodes = contributed.find(
-        Q('category', 'eq', 'project') &
-        Q('is_deleted', 'eq', False) &
-        Q('is_registration', 'eq', False) &
-        Q('is_folder', 'eq', False)
+    nodes = Node.find_for_user(
+        user,
+        subquery=(
+            Q('category', 'eq', 'project') &
```

Comment:
And yes, I recognize that this is entirely out of scope for this PR. But if we agree that it should be done but believe it shouldn't be done here, it would be good to have an issue made for it.